frontend/src/app/main/ui/ds/buttons/_buttons.scss

```diff
@@ -20,14 +20,21 @@
   --button-focus-inner-ring-color: initial;
   --button-focus-outer-ring-color: initial;
 
+  --button-width: initial;
+  --button-height: #{$sz-32};
+  --button-padding: #{$sz-4};
```

Comment:
We don't need to set the padding if we set both `width` and `height` (the component has a `display:flex` that centers its contents), so we can remove this component token.